CodeEdit/Features/Settings/Pages/TextEditingSettings/Models/TextEditingSettings.swift

```diff
@@ -79,40 +77,6 @@ extension SettingsData {
                 BracketPairHighlight.self,
                 forKey: .bracketHighlight
             ) ?? BracketPairHighlight()
-
-            self.populateCommands()
-        }
-
-        /// Adds toggle-able preferences to the command palette via shared `CommandManager`
-        private func populateCommands() {
-            let mgr = CommandManager.shared
-
-            mgr.addCommand(
-                name: "Toggle Type-Over Completion",
-                title: "Toggle Type-Over Completion",
-                id: "prefs.text_editing.type_over_completion",
-                command: CommandClosureWrapper {
-                    Settings.shared.preferences.textEditing.enableTypeOverCompletion.toggle()
-                }
-            )
-
-            mgr.addCommand(
-                name: "Toggle Autocomplete Braces",
-                title: "Toggle Autocomplete Braces",
-                id: "prefs.text_editing.autocomplete_braces",
-                command: CommandClosureWrapper {
-                    Settings.shared.preferences.textEditing.autocompleteBraces.toggle()
-                }
-            )
-
-            mgr.addCommand(
-                name: "Toggle Word Wrap",
-                title: "Toggle Word Wrap",
-                id: "prefs.text_editing.wrap_lines_to_editor_width",
-                command: CommandClosureWrapper {
-                    Settings[\.textEditing].wrapLinesToEditorWidth.toggle()
-                }
-            )
```

Comment:
Are these commands just no longer available from the overlay? It seemed like from the Discord discussion the CommandManager API would stay and the commands registered there would just be combined with the menu bar commands for the overlay.

Reply:
I removed these, as I expect them to be added to the menu bar, which would also add them to the command palette. That said, I probably should've discussed this before removing them. I also removed the CommandManager class, as there were runtime issues with it (purple triangles) and some things would need to be rethought if we would integrate it with extensions